contrib/firebaseio/src/test/resources/testdata.json

```diff
@@ -0,0 +1,302 @@
+    "isActive": true,
+    "balance": "$3,579.20",
+    "picture": "http://placehold.it/32x32",
+    "age": 20,
+    "eyeColor": "green",
+    "name": {
+      "first": "Lillie",
+      "last": "Glenn"
+    },
+    "company": "VISUALIX",
+    "email": "lillie.glenn@visualix.info",
+    "phone": "+1 (844) 476-3891",
+    "address": "127 Kingston Avenue, Roderfield, Arizona, 5033",
+    "about": "Officia tempor eu in voluptate. Ipsum nisi ut irure aliqua laborum excepteur cillum. Cillum ea eiusmod duis tempor quis ut. Est aute ipsum mollit consectetur exercitation fugiat pariatur minim.",
+    "registered": "Monday, June 23, 2014 7:10 AM",
+    "latitude": "39.351941",
+    "longitude": "-147.015315",
+    "tags": [
+      "mollit",
+      "occaecat",
+      "et",
+      "non",
+      "est",
+      "deserunt",
+      "esse",
+      "custom_tag"
+    ],
+    "range": [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9
+    ],
+    "friends": [
+      {
+        "id": 0,
+        "name": "Virginia Watkins"
+      },
+      {
+        "id": 1,
+        "name": "Duffy Hancock"
+      },
+      {
+        "id": 2,
+        "name": "Kelsey Hill"
+      }
+    ],
+    "greeting": "Hello, Lillie! You have 6 unread messages.",
+    "favoriteFruit": "strawberry"
+  },
+  {
+    "_id": "55e61a30fc1bed5d41139b57",
+    "index": 4,
+    "guid": "4ae2f0d8-4648-4f9c-a0ae-8866c2ff07cb",
+    "isActive": true,
+    "balance": "$1,868.85",
+    "picture": "http://placehold.it/32x32",
+    "age": 30,
+    "eyeColor": "green",
+    "name": {
+      "first": "Hudson",
+      "last": "Blevins"
+    },
+    "company": "DOGNOSIS",
+    "email": "hudson.blevins@dognosis.net",
+    "phone": "+1 (808) 407-3340",
+    "address": "554 Portland Avenue, Abrams, Illinois, 6182",
+    "about": "Pariatur elit tempor ut officia minim aliquip anim sint mollit. Nostrud fugiat exercitation irure veniam reprehenderit ad in consequat quis ea nisi. Eiusmod nisi nostrud consectetur nulla eiusmod ipsum excepteur do irure consectetur culpa velit fugiat. Nostrud sit ullamco nostrud eu exercitation labore. Quis magna laboris et consequat. Sit fugiat nisi amet sint culpa ullamco.",
+    "registered": "Friday, December 5, 2014 5:28 AM",
+    "latitude": "0.694342",
+    "longitude": "104.825617",
+    "tags": [
+      "mollit",
+      "occaecat",
+      "et",
+      "non",
+      "est",
+      "deserunt",
+      "esse",
+      "custom_tag"
+    ],
+    "range": [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9
+    ],
+    "friends": [
+      {
+        "id": 0,
+        "name": "Virginia Watkins"
+      },
+      {
+        "id": 1,
+        "name": "Duffy Hancock"
+      },
+      {
+        "id": 2,
+        "name": "Kelsey Hill"
+      }
+    ],
+    "greeting": "Hello, Hudson! You have 8 unread messages.",
+    "favoriteFruit": "strawberry"
+  }
```

Comment:
At first I was like, what is going on with all this PII? Then I realized this must be autogenerated. Nice.